nex/secure.go

```diff
@@ -3,30 +3,41 @@ package nex
 import (
 	"fmt"
 	"os"
+	"strconv"
 
-	nex "github.com/PretendoNetwork/nex-go"
+	nex "github.com/PretendoNetwork/nex-go/v2"
 	"github.com/PretendoNetwork/pokken-tournament/globals"
 )
 
 func StartSecureServer() {
-	globals.SecureServer = nex.NewServer()
-	globals.SecureServer.SetPRUDPVersion(1)
-	globals.SecureServer.SetPRUDPProtocolMinorVersion(3)
-	globals.SecureServer.SetDefaultNEXVersion(nex.NewNEXVersion(3, 10, 0))
-	globals.SecureServer.SetKerberosPassword(globals.KerberosPassword)
-	globals.SecureServer.SetAccessKey("6ef3adf1")
+	globals.SecureServer = nex.NewPRUDPServer()
 
-	globals.SecureServer.On("Data", func(packet *nex.PacketV1) {
-		request := packet.RMCRequest()
+	globals.SecureEndpoint = nex.NewPRUDPEndPoint(1)
+	globals.SecureEndpoint.IsSecureEndPoint = true
+	globals.SecureEndpoint.ServerAccount = globals.SecureServerAccount
+	globals.SecureEndpoint.AccountDetailsByPID = globals.AccountDetailsByPID
+	globals.SecureEndpoint.AccountDetailsByUsername = globals.AccountDetailsByUsername
+	globals.SecureServer.BindPRUDPEndPoint(globals.SecureEndpoint)
+
+	globals.SecureServer.LibraryVersions.SetDefault(nex.NewLibraryVersion(3, 10, 0))
```

Comment:
You also have to add a line for using the structure haeder from NEX 3.5+

```suggestion
	globals.SecureServer.LibraryVersions.SetDefault(nex.NewLibraryVersion(3, 10, 0))
	globals.SecureServer.ByteStreamSettings.UseStructureHeader = true
```

This would be on both auth and secure

Reply:
Oops, forgot about that, added